CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Remove `configmapprovider.NewDefault`, `configmapprovider.NewExpand`, `configmapprovider.NewMerge` (#4600)
 - Move `configtest.LoadConfig` and `configtest.LoadConfigAndValidate` to `servicetest` (#4606)
 - Builder: Remove deprecated `include-core` flag (#4616)
+- Add an option to allow `config.Map` conversion in the `service.ConfigProvider` (#4634)
```

Comment:
Is this a breaking change given that NewDefaultConfigProvider got optional arguments?

Reply:
Nope :D